knative-operator/pkg/common/service_monitor.go

```diff
@@ -0,0 +1,77 @@
+package common
+
+import (
+	"golang.org/x/net/context"
```

Comment:
I don't think this is the correct import. It should be just `"context"`

Reply:
ok my IDE played tricks on my will update.